worker/firewaller/firewaller_test.go

```diff
@@ -234,7 +239,8 @@ func (s *InstanceModeSuite) newFirewallerWithClock(c *gc.C, clock clock.Clock) w
 		NewCrossModelFacadeFunc: func(*api.Info) (firewaller.CrossModelFirewallerFacadeCloser, error) {
 			return s.crossmodelFirewaller, nil
 		},
-		Clock: s.clock,
+		Clock:         s.clock,
+		CredentialAPI: s.credentials,
```

Comment:
Nothing I can see in the tests uses the CredentiaAPI yet. Will that come later?

Reply:
I can add a test somewhere to use it, possibly in the common where the context is created. Since this is essentially a callback, nothing in this layer will actually use it... itt is just passed on to the provider calls.
All we can check here is that it is present.